go.mod

```diff
@@ -169,7 +169,7 @@ require (
 	gopkg.in/warnings.v0 v0.1.2 // indirect
 	gopkg.in/yaml.v2 v2.4.0 // indirect
 	gopkg.in/yaml.v3 v3.0.1 // indirect
-	k8s.io/apiextensions-apiserver v0.26.1 // indirect
+	k8s.io/apiextensions-apiserver v0.26.1
```

Comment:
Direct dependency should not be in indirect dependencies block

Reply:
Huh, I thought `go mod tidy` was supposed to handle this. It removed the `// indirect` comment, but didn't move the dependency.

Moved it manually